api/src/pcapi/routes/backoffice/offers/blueprint.py

```diff
@@ -661,4 +661,22 @@ def get_offer_details(offer_id: int) -> utils.BackofficeResponse:
     if not offer:
         raise NotFound()
 
-    return render_template("offer/details.html", offer=offer, active_tab=request.args.get("active_tab", "stock"))
+    return render_template(
+        "offer/details.html",
+        offer=offer,
+        active_tab=request.args.get("active_tab", "stock"),
+        index_offer_form=empty_forms.EmptyForm(),
+    )
+
+
+@list_offers_blueprint.route("/<int:offer_id>/index_offer", methods=["POST"])
+@utils.permission_required(perm_models.Permissions.ADVANCED_PRO_SUPPORT)
+def index_offer(offer_id: int) -> utils.BackofficeResponse:
+    search.async_index_offer_ids({offer_id})
+
+    flash("La resynchronisation a été lancée avec succès.", "success")
```

Comment:
En fait elle n'a pas encore été lancée, car c'est asynchrone, mais mise dans la file des synchros.
Je suggère :
```suggestion
    flash("La resynchronisation de l'offre a été demandée.", "success")
```

Reply:
Changé